### ISSUE TITLE
Add copy acceleration support to Windows with robocopy

### ISSUE THREAD

**Issue by Korb** [doublecmd/doublecmd#448]
Currently, when copying or transferring a large number (hundreds of thousands) of files over a local network, I have to use archivers, mounting remote storages, physical transfer of disks. For such situations, there are applications like TeraCopy, FastCopy and GS RichCopy 360. However, it is not always clear before starting the migration how appropriate it is to use them instead of the default Double Commander.

Help:
https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy
https://github.com/MicrosoftDocs/windowsserverdocs/issues?q=robocopy

PS. As an alternative way, you can consider the possibility of your own implementation of multi-threaded copying of small files when the main time costs are spent on auxiliary operations, and not on the data transfer itself.

**Comment by Skif-off** [comment]
Why not buttons? The external application is engaged in a long difficult task, and user is engaged in his business, without braking/freezing... For example,
```
command: FastCopy.exe
parameters: %"0 /cmd=noexist_only /auto_close /verify /open_window /estimate /srcfile_w="%LW" /to="%Dt"
```

**Comment by Korb** [comment]
> Why not buttons?

Firstly, I don't understand how the code for the button should look like in the case of robocopy. And secondly, I won't be able to see the progress of each copy operation as conveniently as it is now implemented in Double Commander for regular copy operations.

> The external application is engaged in a long difficult task, and user is engaged in his business, without braking/freezing...

I did not notice any braking or freezing when copying multiple files simultaneously using the standard Double Commander tools. Copy queues are a great feature that I use very often.

> alexoc2000 closed this as completed on Aug 11, 2022

Perhaps this issue should have been closed `as not planned` ("won't fix, can't repro, stale")?
